modules/apps/29-fee/keeper/events_test.go

```diff
@@ -163,7 +163,7 @@ func (suite *KeeperTestSuite) TestDistributeFeeEvent() {
 			sdk.NewAttribute(types.AttributeKeyReceiver, suite.chainA.SenderAccount.GetAddress().String()),
 			sdk.NewAttribute(types.AttributeKeyFee, defaultTimeoutFee.String()),
 		),
-	}
+	}.ToABCIEvents()
```

Comment:
for convenience I left as is and called `ToABCIEvents()` since `sdk.Events` is just a wrapper around []abci.Event